tests/rust_tests.rs

```diff
@@ -1249,16 +1385,31 @@ fn burn_token_test() {
         )
         .assert_ok();
 
+    // [test] check if blockchain reports a new balance of 4
     b_wrapper.check_nft_balance(
         first_user_address,
         SFT_TICKER,
         1u64,
         &rust_biguint!(4),
         Option::<&Empty>::None,
     );
+
+    // [test] burn just 1 of the supply
+    b_wrapper
+        .execute_esdt_transfer(
+            &first_user_address,
+            &setup.contract_wrapper,
+            SFT_TICKER,
+            1u64,
+            &rust_biguint!(5),

```

Comment:
here it should be 4.
the test fails

Reply:
Fixed @bucurdavid , tests passing again - pls review and merge to main